web-animations/interfaces/KeyframeEffect/keyframe-properties.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<meta charset='utf-8'>
+<title>Web Animations API: Keyframe Property tests</title>
+<link rel='help' href='https://w3c.github.io/web-animations/#processing-a-keyframes-argument'>
```

Comment:
This should be merged into web-animations/interfaces/KeyframeEffect/processing-a-keyframes-argument.html